mlflow/server/js/src/components/modals/ConfirmModal.js

```diff
@@ -70,9 +70,9 @@ export class ConfirmModal extends Component {
           </Modal.Title>
         </Modal.Header>
         <Modal.Body>
-          <p style={{ marginBottom: '10px' }}>
```

Comment:
If we're ok with changing this to a div (doesn't seem to change the appearance of other models like run rename, deletion, or restoration) I think this PR could be good to merge as is